README.md

```diff
@@ -42,22 +43,31 @@ Visit the **feedback** section in our [forums](https://forums.improbable.io/) an
 **GitHub issues**</br>
 Create an issue in [this repository](https://github.com/spatialos/UnrealGDK/issues).
 
-## Contributions
-We are not currently accepting public contributions - see our [contributions](https://docs.improbable.io/unreal/pre-alpha/contributing) policy. However, we are accepting issues and we do want your feedback.
-
-## Documentation
-For guidance on using the SpatialOS Unreal GDK, see the [official documentation](https://docs.improbable.io/unreal/pre-alpha/index).
+## Where to get the GDK and starter projects
+The GDK and its starter projects are available on GitHub.
+* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter)
+* [Starter Project](https://github.com/spatialos/UnrealStarterProject)
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
+* [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.19-SpatialOSUnrealGDK) </br>
+**NOTE:** This link may give you a 404. 
+In order to get access to this fork, you need to link your GitHub account to a verified Epic Games account, and to have agreed to Epic's license. You will not be able to use the GDK for Unreal without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
 
-For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/13.1/shared/concepts/spatialos). <br/>
+## SpatialOS Documentation
+* For guidance on SpatialOS concepts see the documentation on the [SpatialOS website](https://docs.improbable.io/reference/latest/shared/concepts/spatialos). <br/>
+* For information on the SpatialOS GDK for Unreal, we recommend you use the [GDK documentation on the SpatialOS website](https://docs.improbable.io/unreal/pre-alpha/index), however, you can look at the [documentation on GitHub](https://github.com/spatialos/UnrealGDK/tree/master/docs).
```

Comment:
Is it worth saying _why_ we recommend this? (I just know that I personally don't like being told to do something if nobody tells me why and there seems to be an easier way)

Reply:
I have an explanation on the top of the docs/readme file which this links to here https://github.com/spatialos/UnrealGDK/tree/master/docs. So if you want to know, the answer is a click through. Thinking adding the explanation here as well might be more clutter on an already cluttered page?

Reply:
Cool, makes sense.

Although I reckon it should be a call-out/banner if possible, just to really deter people :)

Reply:
I added a bit of text to make it clearer without being verbose. (I hope!)